frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -178,13 +112,27 @@ export const ImageManifestVulnDetails: React.FC<ImageManifestVulnDetailsProps> =
           </div>
           <div className="col-sm-6">
             <dl className="co-m-pane__details">
-              <DetailsItem label="Registry" obj={props.obj} path="spec.image" />
+              <DetailsItem
+                label={t('container-security~Registry')}
+                obj={props.obj}
+                path="spec.image"
+              />
+              <DetailsItem
+                label={t('container-security~Manifest')}
+                obj={props.obj}
+                path="obj.spec.manifest"
+              >
+                <ExternalLink
+                  text={shortenHash(props.obj.spec.manifest)}
+                  href={quayURLFor(props.obj)}
+                />
+              </DetailsItem>
             </dl>
           </div>
         </div>
       </div>
-      <div className="co-m-pane__body">
-        <ImageVulnerabilitiesTable features={props.obj.spec.features} />
+      <div className="imagevulnerabilitieslist-wrapper">
```

Comment:
```suggestion
      <div className="imagevulnerabilitieslist__wrapper">
```

can even use pkg name i.e `cs-imagevulnerabilitieslist__wrapper`. WDYT